src/main/java/com/elasticbox/jenkins/SlaveConfiguration.java

```diff
@@ -261,25 +261,38 @@ public ListBoxModel doFillBoxVersionItems(@RelativePath("..") @QueryParameter St
             return DescriptorHelper.getBoxVersions(createClient(endpointUrl, token), workspace, box);
         }
 
-        public ListBoxModel doFillProfileItems(@RelativePath("..") @QueryParameter String endpointUrl,
+        public ListBoxModel doFillProfileItems(@RelativePath("..") @QueryParameter String name,
```

Comment:
It is called by Jenkins automatically when the combo needs to be refreshed, and with the annotation we are telling Jenkins where to get the param from (a form field), so there is no calling method to be updated.